code/game/objects/items/storage/boxes.dm

```diff
@@ -761,3 +761,211 @@
 	else if(!isopened)
 		isopened = 1
 		icon_state = "mealpackopened"
+
+//food boxes for storage in bulk
+
+//meat
+/obj/item/storage/box/meat
+	name = "box of meat"
+
+/obj/item/storage/box/meat/fill_preset_inventory()
+	for(var/i = 0, i < 7, i++)
```

Comment:
@`for(var/foo in 1 to X)` is faster and in the standards.

https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STANDARDS.md#in-to-for-loops

Reply:
replace it by var/i in 1 to 7 like suggest thanks